syft/formats/common/spdxhelpers/to_format_model_test.go

```diff
@@ -5,17 +5,120 @@ import (
 	"regexp"
 	"testing"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/spdx/tools-golang/spdx"
+	"github.com/spdx/tools-golang/spdx/v2/v2_3"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/anchore/syft/syft/artifact"
 	"github.com/anchore/syft/syft/file"
 	"github.com/anchore/syft/syft/pkg"
 	"github.com/anchore/syft/syft/sbom"
+	"github.com/anchore/syft/syft/source"
 )
 
-// TODO: Add ToFormatModel tests
+func Test_toFormatModel(t *testing.T) {
```

Comment:
this should be a completion test, ensuring it's covering all source types https://github.com/anchore/syft/blob/e1c1832f8413de04f9c6a47c1f22b3a526473231/syft/internal/sourcemetadata/completion_tester.go#L8